packages/zoe/src/issuerStorage.js

```diff
@@ -1,7 +1,7 @@
+import { deeplyFulfilledObject, objectMap } from '@agoric/internal';
```

Comment:
```suggestion
import { objectMap } from '@endo/common/object-map.js';
import { deeplyFulfilledObject } from '@agoric/internal';
```

And all other occurrences.

Reply:
I see this in 16 other files; can we defer it to a dedicated PR?
```console
$ git grep -l "import.*objectMap.* from '@agoric/internal'" packages/ | tee /dev/stderr | wc -l
packages/SwingSet/test/upgrade/test-upgrade.js
packages/SwingSet/tools/bootstrap-relay.js
packages/agoric-cli/src/commands/inter.js
packages/builders/scripts/inter-protocol/init-core.js
packages/governance/src/contractHelper.js
packages/inter-protocol/test/auction/test-scheduler.js
packages/inter-protocol/test/auction/tools.js
packages/inter-protocol/test/vaultFactory/driver.js
packages/inter-protocol/test/vaultFactory/test-replacePriceAuthority.js
packages/inter-protocol/test/vaultFactory/test-vaultFactory.js
packages/inter-protocol/test/vaultFactory/test-vaultLiquidation.js
packages/smart-wallet/src/offerWatcher.js
packages/smart-wallet/src/utils.js
packages/wallet/api/src/lib-wallet.js
packages/zoe/src/cleanProposal.js
packages/zoe/src/contractFacet/zcfZygote.js
packages/zoe/src/contractSupport/durability.js
packages/zoe/src/issuerStorage.js
packages/zoe/src/zoeService/escrowStorage.js
19
```

Reply:
Sigh. Yes.

Reply:
Resolving.